src/main/java/com/brightcove/zencoder/client/model/DeliveryFormat.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (C) 2013 Brightcove Inc. All Rights Reserved. No use, copying or distribution of this
+ * work may be made except in accordance with a valid license agreement from Brightcove Inc. This
+ * notice must be included on all copies, modifications and derivatives of this work.
+ *
+ * Brightcove Inc MAKES NO REPRESENTATIONS OR WARRANTIES ABOUT THE SUITABILITY OF THE SOFTWARE,
+ * EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, OR NON-INFRINGEMENT. BRIGHTCOVE SHALL NOT BE
+ * LIABLE FOR ANY DAMAGES SUFFERED BY LICENSEE AS A RESULT OF USING, MODIFYING OR DISTRIBUTING THIS
+ * SOFTWARE OR ITS DERIVATIVES.
+ *
+ * "Brightcove" is a registered trademark of Brightcove Inc.
+ */
+package com.brightcove.zencoder.client.model;
+
+/**
+ * @see https://app.zencoder.com/docs/api/encoding/segmented-streaming/streaming-delivery-format
+ */
+public enum DeliveryFormat {
```

Comment:
This seemed like the appropriate name for the class, following the standards of other names, and the JSON reference of `streaming-delivery-format`.